blocks/core/ff_module/ff_module-task-event/src/TaskEventContainer.rt

```diff
@@ -0,0 +1,6 @@
+<div
```

Comment:
There are lots of containers in the patternlib though this doesn't entirely fit the convention. Perhaps a synonym like wrapper or something like base?